Resources/config/services.xml

```diff
@@ -14,6 +14,11 @@
 
         <service id="Ekino\DataProtectionBundle\Encryptor\EncryptorInterface" alias="ekino_data_protection.encryptor" />
 
+        <service id="Ekino\DataProtectionBundle\Command\EncryptPasswordCommand" class="Ekino\DataProtectionBundle\Command\EncryptPasswordCommand" public="true">
+            <argument>%kernel.secret%</argument>
```

Comment:
please don't inject this parameter, but the configured value (and so for cipher method)